### ISSUE TITLE
Marketing strategies

### ISSUE THREAD

**Issue by sschiessl-bcp** [bitshares/marketing#11]
This is an example of good marketing made by Kraken exchange
![image](https://user-images.githubusercontent.com/33128181/61047387-aef77c00-a3df-11e9-8300-427d5110475b.png)

**Comment by bitcoinsfacil** [comment]
Correct, that is an example of a good conversion funnel @sschiessl-bcp it segments clearly what it can offer to the person who `lands` on your website. Is this a way of gathering proposal to improve current landings on BitShares?
